module2/lessons/how_the_web_works_dns.md

```diff
@@ -27,13 +27,19 @@ We know that in the HTTP request/response cycle, there are two players. A client
 
 When you send a letter to a friend, you have to specify their address. This address uniquely identifies your friend's house in the world. Computers have a corresponding address called an **IP Address**. The IP address address uniquely identifies a device on the internet. IP stands for "Internet Protocol". In order to find a computer anywhere in the world, all you need is its IP address, for example 142.811.0.1.
 
-However, when you go to your web browser, you don't type in "142.811.0.1", you type in "google.com". So how does "google.com" become "142.811.0.1"? The answer is DNS.
+However, when you go to your web browser, you don't type in "142.811.0.1", you type in "google.com". So how does "google.com" become "142.811.0.1"? The answer is **DNS**.
 
 ## DNS
 
-DNS is the Domain Name System. It's entire job is to take a domain, like google.com, and translate it to an IP address, like 142.811.0.1.
+DNS is the Domain Name System. Its entire job is to take a domain, like `google.com`, and translate it to an IP address, like 142.811.0.1.
 
-You can think of DNS as the phonebook of the internet.
+You can think of DNS as the phonebook (or, in more modern terms a *contact list*) of the internet. While you might click "Juliet" in your contact list to call Juliet's phone, the phone company actually connects to `1-123-555-1234` since that's Juliet's phone number. 
+
+Similarly, when we go to `https://google.com`, DNS is the translation between what that website is *called* and what it actually *is* to the internet. So, if we were to express this correlation as an analogy: 
+
+Contacts : Domain Names :: Phone Numbers : IP Addresses
+
+You can look up DNS information for your favorite websites on a Mac by using the `dig` command in Terminal. For example, type `dig google.com` and see what it comes back with in the `ANSWER` section. Can you identify the IP address that matches that domain? 
```

Comment:
Maybe we should give them instructions on how to find their IP address so that they are able to make the connection when looking at the `ANSWER` section.